flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/JarHandlerParameterTest.java

```diff
@@ -211,6 +218,37 @@ void testProvideJobId() throws Exception {
         assertThat(jobGraph.get().getJobID()).isEqualTo(jobId);
     }
 
+    @Test
+    void testProvideFlinkConfig() throws Exception {
+
+        REQB jarRequestBodyWithFlinkConfig = getJarRequestBodyWithFlinkConfig(FLINK_CONFIGURATION);
+        HandlerRequest<REQB> request =
+                createRequest(
+                        getJarRequestBodyWithFlinkConfig(FLINK_CONFIGURATION),
+                        getUnresolvedJarMessageParameters(),
+                        getUnresolvedJarMessageParameters(),
+                        jarWithManifest);
+
+        handleRequest(request);
+
+        Optional<JobGraph> jobGraph = getLastSubmittedJobGraphAndReset();
+
+        assertThat(jobGraph.isPresent()).isTrue();
+        JobGraph graph = jobGraph.get();
+
+        assertThat(getExecutionConfig(graph).getParallelism())
+                .isNotEqualTo(
+                        Integer.valueOf(
+                                FLINK_CONFIGURATION.get(CoreOptions.DEFAULT_PARALLELISM.key())));
+        if (jarRequestBodyWithFlinkConfig instanceof JarRunRequestBody) {
```

Comment:
Why is this specific to the jar run handler?

Reply:
becasue just jarRunHandler can set flinkConfig,this test is include jobPlanhandler

Reply:
It is actually a bit questionable to have this only work for the JarRunHandler; the 2 handlers should generally have the same capabilities; the whole point of the JarPlanHandler is to get a preview of the job after all.

Reply:
You should also add the flinkConfig logic to the JobPlanHandler

Reply:
Do I need to open a new issuse to complete this, or continue to complete this function on this issuse?

Reply:
Now I've added the logic of the JobPlanHandler part